esp/src/eclwatch/TopologyWidget.js

```diff
@@ -18,53 +18,163 @@ define([
+                        }
+                            WsMachine.GetMachineInfo ({
+                                request: {
+                                    [MachineInformationClean]: filter[MachineInformationClean],
+                                    [MachineInformationCount]: 1000,
+                                    Path: selection[i].__hpcc_treeItem.Path,
+                                    Cluster: selection[i].__hpcc_treeItem.Name,
+                                    AutoRefresh:filter.AutoRefresh,
+                                    MemThreshold:filter.MemThreshold,
+                                    DiskThreshold:filter.DiskThreshold,
+                                    CpuThreshold:filter.CpuThreshold,
+                                    MemThresholdType:filter.MemThreshold,
+                                    DiskThresholdType:filter.DiskThresholdType,
+                                    GetProcessorInfo:filter.GetProcessorInfo,
+                                    GetStorageInfo:filter.GetStorageInfo,
+                                    LocalFileSystemsOnly:filter.LocalFileSystemsOnly,
+                                    GetSoftwareInfo:filter.GetSoftwareInfo,
+                                    DiskThreshold:filter.DiskThreshold,
+                                    DiskThresholdType:filter.DiskThresholdType,
+                                    ApplyProcessFilter:filter.ApplyProcessFilter,
+                                    AddProcessesToFilter: filter.AddtionalProcessesToFilter,
+                                    cbAutoRefresh: filter.cbAutoRefresh,
+                                }
+                            }).then(function (response) {
+                                var pfTab = context.ensureMIPane(response.GetMachineInfoResponse.Machines.MachineInfoEx[0].Address, {
+                                    params: response.GetMachineInfoResponse
+                                });
+                                context.detailsWidget.requestInformationWidget.set("disabled", false);
+                                context.detailsWidget.requestInformationWidget.init(response.GetMachineInfoResponse);
+                                pfTab.init(response.GetMachineInfoResponse, "machines");
+                            });
+                    } else {
+                        var TargetClustersClean = "TargetClusters." + [i];
+                        var TargetClusterCount = "TargetClusters.itemcount";
+                        filter[TargetClustersClean] = selection[i].__hpcc_treeItem.Type + ":" + selection[i].__hpcc_treeItem.Name;
+                        WsMachine.GetTargetClusterInfo ({
+                            request: {
+                                [TargetClustersClean]: filter[TargetClustersClean],
+                                [TargetClusterCount]: 1000,
```

Comment:
I verified with @wangkx this count seems to increase based on selections and is subjective.